rest-api/config/config_stable.json

```diff
@@ -6,10 +6,14 @@
   ],
   "user_info": {
     "pmi-hpo-test@appspot.gserviceaccount.com": {
-      "roles": ["ptc", "healthpro"]
+      "roles": ["ptc", "healthpro"],
```

Comment:
While we're updating this, I don't think there's a reason for this user to have the `ptc` role.

Reply:
Removed.

(In the distant past, HPRO could create participants via a form and needed to act like PTC, which I'm guessing is why this was here. I presume we don't want to support such shenanigans anymore. :)